README.rst

```diff
@@ -15,8 +15,12 @@ An implementation of wagtailsurveys as a Molo plugin
 
 Installation::
 
-   pip install molo.surveys
+   pip install -e ../molo.surveys for local package installs
+   pip install ../molo.surveys for global package installs
```

Comment:
I think this should be `pip install molo.surveys to install from PyPI`
and leave out the local install one. It's a relative path and whoever is reading this might have their local repository in a different place or directory.